website/app/controllers/show.js

```diff
@@ -116,6 +135,9 @@ export default class ShowController extends Controller {
     this.tocs.forEach((toc) => {
       set(toc, 'isCurrent', toc.index === current);
     });
+
+    set(this, 'selectedTab', this.tabs[current].label.toLowerCase());
```

Comment:
all these string manipulations scattered around the code, to be used as identifiers, make me nervous. better to use an ID or an ad-hoc `data-` attribute

Reply:
it _is_ limited to the two instances in this file and has tests around it all, and it would help us return the correct tab regardless of how a consumer happens to capitalize the query param (unlikely, but possible). I do see your point though, so I'll see if there is a way to  address this more consistently and/or at least make comments explicitly noting this.

Reply:
added a comment for the two instances of this